hooks/useCollections.ts

```
@@ -19,25 +21,47 @@ async function fetchCollections(
+      staleTime: 1000 * 60 * 5,
+      refetchOnWindowFocus: false,
+      retry: (failureCount) => {
+        return failureCount < 4;
+      },
+    });
+
+  useEffect(() => {
+    if (data?.pages.length) {
+      setCollections(
+        data?.pages?.reduce((acc: Collection[], page) => {
+          return [...acc, ...page.collections];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)